### PR TITLE
Vscode pr - split 2 - update conda env

### PR DESCRIPTION
**Pull request by Pesc0**

### CODE REVIEW

File: conda/environment.devenv.yml [hunk 1]
```diff
@@ -90,11 +91,10 @@ dependencies:
...
-- smesh==9.9
+- smesh
 - swig
 - vtk
 - xerces-c
 - yaml-cpp
 - zlib
-- pip:
-  - ptvsd
+- debugpy
```
Comment (adrianinsaval) [inline conda/environment.devenv.yml:100]: debugpy is duplicated
```suggestion
```

**Comment by adrianinsaval** [comment]
for once 😆

**Comment by Pesc0** [comment]
freecad-feedstock needs to reflect these changes: see Jacob's comment here https://github.com/FreeCAD/FreeCAD/pull/9663#issuecomment-1778175176

Ideally the two feedstock repos use the conda env file from this repo as a single source of truth.

**Comment by Pesc0** [comment]
@adrianinsaval sorry for the ping but I'm not sure if you've seen this.

**Pull request merged**